### ISSUE TITLE
SMTP port blocked by list monk

### ISSUE THREAD

**Issue by filio123321** [knadh/listmonk#1136]
**Version:**
 - listmonk:  v2.3.0
 - OS: Debian 11

**Description of the bug and steps to reproduce:**
I just installed installed the service and ran it with docker compose and tried to change the setting with my smtp server but I got timeout. after nmapping my smtp from the server that runs listmonk and another which has nothing to do with it, found out that for some reason listmonk blocks ports 465 and 587. When trying to map specifically ports 465 ad 587, it returns them as "filtered" so they work, but the system blocks them. Server is fresh, never ran anything on it before listmonk.

I could't find another issue similar too this and I'm sorry if this is something I caused, but would really appreciate help if possible!

**Comment by filio123321** [comment]
Nvm it was Linode blocking all these ports trying to fight spam
